src/js/veteran-id-card/containers/VeteranIDCard.jsx

```diff
@@ -5,9 +5,10 @@ import RequiredLoginView from '../../common/components/RequiredLoginView';
 import RequiredVeteranView from '../components/RequiredVeteranView';
 import EmailCapture from './EmailCapture';
 
-const rateLimitAuthed = window.settings.vic.rateLimitAuthed;
-const rateLimitUnauthed = window.settings.vic.rateLimitUnauthed;
-const serviceRateLimitedRandomizer = Math.random();
+const vicSettings = {
+  serviceRateLimitedUnauthed: Math.random() > window.settings.vic.rateLimitUnauthed,
+  serviceRateLimitedAuthed: window.sessionStorage.getItem('vicInitialAuthStatus') !== 'unauthenticated' && Math.random() > window.settings.vic.rateLimitAuthed
+};
 
```

Comment:
This is not pretty but I felt like it was getting too cluttered and therefore counterproductive to define a bunch of constants that will ultimately only be used for determining the `serviceRateLimitedUnauthed` and the `serviceRateLimitedAuthed` booleans.